Build/QIR_validation.ps1

```diff
@@ -0,0 +1,118 @@
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples algorithms order-finding OrderFinding.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples algorithms repeat-until-success RepeatUntilSuccess.csproj),
+    (Join-Path $PSScriptRoot .. samples algorithms reversible-logic-synthesis ReversibleLogicSynthesis.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples algorithms simple-grover SimpleGroverSample.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples algorithms sudoku-grover SudokuGroverSample.csproj),
+
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples azure-quantum grover Grover.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples azure-quantum hidden-shift HiddenShift.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples azure-quantum ising-model IsingModel.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples azure-quantum parallel-qrng ParallelQrng.csproj),
+    # package version issue #(Join-Path $PSScriptRoot .. samples azure-quantum teleport Teleport.csproj),
+
+    (Join-Path $PSScriptRoot .. samples characterization phase-estimation PhaseEstimationSample.csproj),
+
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry AnalyzeHamiltonian 1-AnalyzeHamiltonian.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry CreateHubbardHamiltonian CreateHubbardHamiltonian.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry GetGateCount 3-GetGateCount.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry MolecularHydrogen MolecularHydrogen.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry RunSimulation 2-RunSimulation.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples chemistry SimulateHubbardHamiltonian SimulateHubbardHamiltonian.csproj),
+
+    (Join-Path $PSScriptRoot .. samples error-correction bit-flip-code BitFlipCode.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples error-correction syndrome Syndrome.csproj),
+
+    (Join-Path $PSScriptRoot .. samples getting-started measurement Measurement.csproj),
+    (Join-Path $PSScriptRoot .. samples getting-started qrng Qrng.csproj),
+    # needs argument(s) #(Join-Path $PSScriptRoot .. samples getting-started simple-algorithms SimpleAlgorithms.csproj),
+    (Join-Path $PSScriptRoot .. samples getting-started teleportation TeleportationSample.csproj),
+
+    # not compatible #(Join-Path $PSScriptRoot .. samples machine-learning half-moons HalfMoons.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples machine-learning parallel-half-moons ParallelHalfMoons.csproj),
+    # not compatible #(Join-Path $PSScriptRoot .. samples machine-learning wine Wine.csproj),
+
+    # package version issue #(Join-Path $PSScriptRoot .. samples numerics CustomModAdd CustomModAdd.csproj),
```

Comment:
What is the package version issue?

Reply:
I went back to recreate the issues to give you some specifics to answer your question, and found that I couldn't recreate the issues. I've removed those designations and included those samples. The package version might just be something to keep an eye on as this becomes a validation step in the CI.